public/js/include/bootstrap.min.js

```diff
@@ -0,0 +1,7 @@
+/*!
```

Comment:
Actually we dont even need this because bootstrap is included in the framework by default